src/commands/ban.rs

```diff
@@ -51,15 +51,22 @@ async fn do_ban(
         .await
         .context("Failed to retrieve user for banned user")?;
 
-    let _ = user
-        .dm(&ctx, |m| {
-            m.embed(|e| {
-                e.title(format!("You where banned from {}", guild.name));
-                e.field("Reason", reason, false)
+    if reason.to_string().contains("ice") {
+        let _ = user
+            .dm(&ctx, |m| -> &mut serenity::builder::CreateMessage {
+               m.content("Hey ice, you where banned once again. Instead of wasting your time spamming here, please consider seeking help regarding your mental health.\nhttps://www.nimh.nih.gov/health/find-help/index.shtml")
```

Comment:
You might want to change 'you where banned' to 'you were banned'

Reply:
My bad, I just copy pasted what elkowar gave me :pensive:
I'll fix that later when I'm on my laptop

Reply:
@Tanish2002 you to fix this

Reply:
done UwU